Source/GUI/BuildWindow.cs

```diff
@@ -809,9 +809,10 @@ void OnFlagSelected (FlagBrowser.FlagEntry selected)
 		private void craftSelectComplete (string filename,
 										  CraftBrowserDialog.LoadType lt)
 		{
-			control.LoadCraft (filename, flagURL);
 			control.craftType = craftlist.craftType;
 			craftlist = null;
+
+			control.LoadCraft(filename, flagURL);
```

Comment:
Does this change actually have a purpose? I can't see any any issue with it being either way, just curious.